src/psyclone/psyir/nodes/operation.py

```diff
@@ -197,7 +197,9 @@ class BinaryOperation(Operation):
         # Other Maths Operators
         'SIGN', 'MIN', 'MAX',
         # Query Operators
-        'SIZE'
+        'SIZE',
+        # Matrix and Vector Operators
```

Comment:
We really need to precisely document these various intrinsics that we support - what arguments (and types) they take and what they return. However, I'm attempting to find a nice way to do that in #672 so I'll let you off here.

Reply:
Whoop :-)